### PR TITLE
rm: add an additional flag -R for --recursive 

### PR DESCRIPTION
**Pull request by jhscheer**
make clap support -R in addition to -r

### CODE REVIEW

File: src/uu/rm/src/rm.rs [hunk 1]
```diff
@@ -128,6 +128,10 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             .long(OPT_RECURSIVE)
             .help("remove directories and their contents recursively")
         )
+        .arg(
```
Comment (sylvestre) [inline src/uu/rm/src/rm.rs:131]: I think you could use the --alias option ?
https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.alias

Comment (jhscheer) [reply src/uu/rm/src/rm.rs:131]: I also thought clap's `visible_alias` function could be use for this. However, I as far as I can tell it is not possible to mimic GNU's behaviour with `alias`. The problem is that alias is only for the long flag, i.e. with `.visible_alias("R")` the user would have to type `--R`.

Comment (sylvestre) [reply src/uu/rm/src/rm.rs:131]: OK, TIL :)

Could you please add a comment to make it explicit in the code? thanks

Comment (jhscheer) [reply src/uu/rm/src/rm.rs:131]: done

**Review by sylvestre — APPROVED** [review]

**Pull request merged**